index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/app.css">
+    <title>Emir Sağıt Personal Website</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="nav-flex">
+                <a href="#" class="nav-text">My Logo</a>
+                <button class="nav-button" type="button">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" stroke="#6070FF" viewBox="0 0 24 24">
+                        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+                    </svg>
+                </button>
+            </div>
+        </nav>
```

Comment:
- [ ] The header tag must wrap this section and no more

Reply:
Well addressed the issue